docs/basic_installation.md

```diff
@@ -0,0 +1,95 @@
+# Basic Installation
+
+This is the easiest way to install the operator and suitable for most users.
+
+This will install/deploy the operator with the most recent version and without
+changing any resource created by the CNWAN Owners.
```

Comment:
I can probably rephrased this `yaml resources` to make it better understandable.

Reply:
Ah, ok. In that case, to make it extra clear, I would not mention "CNWAN Owners", I would just say "install/deploy the latest released version of the operator with the unmodified YAML resources from this repository." I wanted to add ** released** version, because the master can be thought of as version too, which is a moving target.

Reply:
Yes, I agree. It makes more sense like that